src/components/Button/index.tsx

```diff
@@ -21,6 +21,7 @@ interface IButtonProps {
   children?: React.ReactNode
   icon?: ReactElement
   testId?: string
+  style?: React.HTMLAttributes<HTMLButtonElement>["style"]
```

Comment:
So, this becomes useless.

Reply:
Again, this is me rushing this feature in. Sorry, I tried doing this the tailwind way, after 2-3 times I tried stuff I gave up and just passed in a `style` prop...

I think I tried this tool: https://transform.tools/css-to-tailwind

Which gave me this:

```
-mt-2.5 mb-0 -mx-2.5 pt-2.5 pb-0 px-2.5 border-b-[#666] border-b border-solid;
```

Or something like that. I think the `[#666]` part didn't work.

I may be confusing this with another component, but believe me, I gave it a shot and it didn't go easily.

Tailwind is an interesting choice, I just couldn't find the time to go into it deeper.